### PR TITLE
chap13

### PR DESCRIPTION
**Pull request by Lysiek**

### CODE REVIEW

**Comment by Lysiek** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -19,9 +19,16 @@ gem "sqlite3"
 gem "bcrypt"
 gem "config"
 gem "faker", "1.7.3"
+gem 'carrierwave',             '1.2.2'
```
Comment (DuongNT1512) [inline Gemfile]: double quote
convention

**Comment by Lysiek** [comment]
ready

File: Gemfile [hunk 2]
```diff
@@ -19,9 +19,16 @@ gem "sqlite3"
 gem "bcrypt"
 gem "config"
 gem "faker", "1.7.3"
+gem "carrierwave",             "1.2.2"
```
Comment (DuongNT1512) [inline Gemfile]: convention

File: Gemfile [hunk 3]
```diff
@@ -19,9 +19,16 @@ gem "sqlite3"
...
+gem "mini_magic",             "4.7.0"
 gem "will_paginate", "3.1.6"
 gem "bootstrap-will_paginate", "1.0.0"
 
+group :production do
+  gem "pg",  "0.20.0"
```
Comment (DuongNT1512) [inline Gemfile]: convention

**Comment by Lysiek** [comment]
ready

**Pull request merged**